src/Magnum/DebugTools/CMakeLists.txt

```
@@ -72,7 +72,7 @@ if(TARGET_GL)
         endif()
     endif()
 
-    if(NOT MAGNUM_TARGET_WEBGL)
+    if(NOT MAGNUM_TARGET_WEBGL OR NOT MAGNUM_TARGET_GLES2)
```

The following seems like a less confusing condition to me. Same elsewhere:

```suggestion
    if(NOT (MAGNUM_TARGET_WEBGL AND MAGNUM_TARGET_GLES2)
```